.github/workflows/docs.yml

```diff
@@ -0,0 +1,57 @@
+name: Docs
+on:
+  pull_request:
+    types: [synchronize, edited, opened]
+  push:
+    branches: [main]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Install Poetry
+        run: pipx install poetry
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          cache: poetry
+      - name: Install Environment
+        run: poetry install --with=docs
+      - name: Build Documentation
+        run: >-
+          poetry run sphinx-build
+          --color
+          -b html
+          ${{github.workspace}}
+          ${{github.workspace}}/_site
+      - name: Archive Documentation
+        run: >-
+          tar
+          --create
+          --gzip
+          --verbose
+          --mode=a+rw
+          --file=cps-docs.tar.gz
+          --directory=${{_site}}
```

Comment:
Oh. Right, um... stupid question, is this really supposed to be `${{_site}}`, or is it supposed to be `_site`? Or maybe `${{github.workspace}}/_site`? I wonder if you meant to remove the `${{}}` entirely and I, not paying close enough attention, "tricked" you into adding it back... (my bad!)

Reply:
oh you're totally right. it should be `github.workspace`🤦‍♀️